test/generators/java/JavaGenerator.spec.ts

```diff
@@ -342,6 +341,74 @@ describe('JavaGenerator', () => {
       expect(models.map((model) => model.result)).toMatchSnapshot();
     });
 
+    test('should create an interface for child models', async () => {
```

Comment:
perhaps it's enough to only have a test with Jackson preset?

Reply:
Sure. Probably overkill on the testing scenarios.

Reply:
Removed